include/wx/animate.h

```diff
@@ -19,6 +19,7 @@
 #include "wx/control.h"
 #include "wx/timer.h"
 #include "wx/bitmap.h"
```

Comment:
```suggestion
```
(for consistency, we should remove existing inclusions of `wx/bitmap.h` when adding includes of `wx/bmpbndl.h` everywhere, not just in some places).